GameServerLib/GameObjects/Other/CollisionHandler.cs

```diff
@@ -29,7 +30,8 @@ public CollisionHandler(IMap map)
                 // Subtract one cell's size from the max so we never reach the CellCountX/Y (since Cells is an array).
                 _map.NavigationGrid.MaxGridPosition.X + System.MathF.Abs(_map.NavigationGrid.MinGridPosition.X),
                 _map.NavigationGrid.MaxGridPosition.Z + System.MathF.Abs(_map.NavigationGrid.MinGridPosition.Z),
-                new CollisionObject()
+                _objectBounds,
+                byte.MaxValue
```

Comment:
I think that this change actually sort of remove the purpose of QuadTree. This number is the maximum number of objects on one "partition" of the QuadTree and not the maximum objects in general as far as I know. If there is a problem in the collision system it is maybe somewhere else (probably it is related to the upside down y axis in general in some of the conversions).